source/production/F0.Minesweeper.Logic/Minefield.cs

```diff
@@ -1,33 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using F0.Minesweeper.Logic.Abstractions;
+using F0.Minesweeper.Logic.Minelayer;
 
 namespace F0.Minesweeper.Logic
 {
+	[DebuggerDisplay("{" + nameof(DebugVisualization) + ", nq}")]
 	internal sealed class Minefield : IMinefield
 	{
 		private readonly uint width, height, mineCount;
+		private IMinelayer mineplacer;
+		private bool isFirstUncover = true;
+		private bool[,] minefield;
+		private string DebugVisualization
+		{
+			get
+			{
+				System.Text.StringBuilder sb = new System.Text.StringBuilder();
```

Comment:
To be in sync with other code, I suggest using `var` (or a _target-typed new expressions_ if you prefer) and shorten the `StringBuilder` type name by adding a `using System.Text;`

Reply:
Yep, thats right. Didn't want to pollute the usings of the class as I thought about deleting this property after using it.
_But_ then I kept it and forgot to clean up 🙈  Thanks!